pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[project]
+name = "unitycatalog-function"
```

Comment:
```suggestion
name = "unitycatalog-functions"
```

should this be plural?

Reply:
btw is this a name space package?

Reply:
Yea this name is pending to be changed, I'm waiting for Xiangrui's response :)